### PR TITLE
Adds checkbox and remove button on data-select-row

### PR DESCRIPTION
**Pull request by stephanwlee**
This feature allows user to easily toggle a data selection group
or remove the group entirely from the comparison.

image

### CODE REVIEW

File: tensorboard/components/tf_data_selector/tf-data-selector.html [hunk 1]
```diff
@@ -63,13 +65,28 @@
...
       }
+
+      tf-data-select-row {
+        margin: 5px 0;
+        max-width: 720px;
+        width: 100%;
+      }
+
+      tf-data-select-row[no-experiment] {
+        margin-left: -8px;
```
Comment (nfelt) [inline tensorboard/components/tf_data_selector/tf-data-selector.html]: I assume the negative margin somehow accounts for the checkbox?  But I'm not sure why it's needed if the checkbox disappears when noExperiment is true already...

File: tensorboard/components/tf_data_selector/tf-data-select-row.ts [hunk 2]
```diff
@@ -31,6 +31,17 @@ Polymer({
       }),
     },
 
+    enabled: {
```
Comment (nfelt) [inline tensorboard/components/tf_data_selector/tf-data-select-row.ts]: I could be missing something, but is this actually functional in this PR?  Nothing seems to be hooked to this property except the checkbox so AFAICT it doesn't actually control the selection.

Speaking of which, is the thought that checkbox enable/disable state will also be persistent via the URL hash?  Or is that transient state only?

File: tensorboard/components/tf_data_selector/tf-data-selector.html (hunk 1 above)
Comment (stephanwlee) [reply tensorboard/components/tf_data_selector/tf-data-selector.html]: Ah, this is about undoing some padding that paper-input naturally has to align _visual_ left edge with other contents

File: tensorboard/components/tf_data_selector/tf-data-select-row.ts (hunk 2 above)
Comment (stephanwlee) [reply tensorboard/components/tf_data_selector/tf-data-select-row.ts]: Correct. It is there visually but does not actually do anything useful. Eventually, we will persist it in the URL (via hash) as you indicated :)

**Pull request merged**